internal/pkg/cli/app_deploy.go

```diff
@@ -414,6 +417,9 @@ func (o *appDeployOpts) deployAppStack(appTemplate *bytes.Buffer, addonsURL stri
 		stack.EnvTagKey:     o.targetEnvironment.Name,
 		stack.AppTagKey:     o.AppName,
 	}
+	for k, v := range o.targetProject.Tags {
```

Comment:
Silly question - but would it be easier to take the targetProject.Tags and just add the stack.ProjectTagKey, EnvTagKey and AppTagKey with their values?

`o.targetProject.Tags[stack.ProjectTagKey] = o.ProjectName()`

Reply:
I don't think I'm following 😛 
Are you trying to invert how tags are applied? I had this order so that user provided tags take precedent over ours.
Eventually, when we introduce application tags in the manifest, we'll need to think of a way of merging project tags and app tags but I was punting on that until we work on it.

Reply:
Yea exactly - just inverting how tags are applied. I kind of like reserving these tags for us - it gives us flexibility to reliably query for resources by Tag Group.

Reply:
Gotcha, would you mind if I do that change in the `stack` package instead of here? It'd have the same effect.

Reply:
No prob!

Reply:
Done!